prev.tfvars

```diff
@@ -0,0 +1,3 @@
+env_type = "prev"
+region = "northeurope"
```

Comment:
try to reshape it to the global default variable depend on environment - like more complicated object in the variables eg. 
```
variable "regions" {
  type = list(object({
    dev = string
    stage = string
    prod = string
  }))
  default = [
    {
      dev = "northeurope"
      stage = "northeurope"
      prod = "northeurope"
    }
  ]
}
```
after that you can use `regions[terraform.workspace]` in place where variable should be used - you can also create more complicated objects like eg. list of objects(objects) eg `config[terraform.workspace].instance.type` - depend on you